pallets/offchain-worker/src/lib.rs

```diff
@@ -99,11 +99,15 @@ pub mod pallet {
 
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		/// The on_initialize don't need computation or DB access
+		/// It just return the weight of on_finalize
```

Comment:
```suggestion
		/// This should return the weight of `on_initialize` + `on_finalize`
		/// Since `on_initialize` don't need computation or DB access
		/// It just returns the weight `of on_finalize`
```